-- Remove build-system dependencies (cmake, cibuildwheel, delocate) that are provided by the ports framework.
--- pyproject.toml.orig	2026-10-10 11:20:08 UTC
+++ pyproject.toml
@@ -40,27 +40,17 @@ requires = [
 
 [build-system]
 requires = [
-    "setuptools>=77.0.3",
-    "pybind11>=2.13.6",
-    "cibuildwheel>=2.19.2,<3.0.0; python_version < '3.11'",
-    "cibuildwheel>=2.19.2,<4.0.0; python_version >= '3.11'",
+    "setuptools>=0",
+    "pybind11>=0",
     "wheel>=0.43.0,<1.0.0",
-    "delocate>=0.11.0,<1.0.0",
-    "cmake>=3.27.0,<5.0.0; sys_platform != 'win32'",
-    "cmake>=4.2.0,<5.0.0; sys_platform == 'win32'"
 ]
 build-backend = "setuptools.build_meta"
 
 [dependency-groups]
 build = [
-    "setuptools>=77.0.3",
-    "pybind11>=2.13.6",
-    "cibuildwheel>=2.19.2,<3.0.0; python_version < '3.11'",
-    "cibuildwheel>=2.19.2,<4.0.0; python_version >= '3.11'",
+    "setuptools>=0",
+    "pybind11>=0",
     "wheel>=0.43.0,<1.0.0",
-    "delocate>=0.11.0,<1.0.0",
-    "cmake>=3.27.0,<5.0.0; sys_platform != 'win32'",
-    "cmake>=4.2.0,<5.0.0; sys_platform == 'win32'"
 ]
 dev = [
     "pytest>=9.0.3,<10.0.0",
